app/src/main/kotlin/edu/artic/AppComponent.kt

```diff
@@ -3,12 +3,12 @@ package edu.artic
 import dagger.Component
 import dagger.android.AndroidInjector
 import dagger.android.support.AndroidSupportInjectionModule
-import edu.artic.main.MainModule
 import edu.artic.db.ApiModule
 import edu.artic.db.DBModule
 import edu.artic.events.EventsModule
+import edu.artic.exhibitions.ExhibitionsModule
+import edu.artic.main.MainModule
```

Comment:
Can you revert these file changes? it'll cause conflicts from @sam33rdhakal branch.

Reply:
Will do so after @sam33rdhakal's changes are put to dev